### PR TITLE
degree

### PR DESCRIPTION
**Pull request by claudiarrese**

### CODE REVIEW

**Review by ASanchezXtillion — CHANGES_REQUESTED** [review]
Change from view to mst json

File: migration_original/ODS1Stage/tables/Base/Degree/spu_translated_Degree.sql [hunk 1]
```diff
@@ -30,22 +30,22 @@ begin
...
-                            where d.degreeabbreviation = json.degree_DegreeCode
-                        ) and DegreeCode is not null
+                    with Cte_degree as (
+                        select
+                            distinct
+                            to_varchar(json.value:DEGREE_CODE) as DegreeCode,
+                            MAX(to_varchar(json.value:UPDATED_DATETIME)) as LastUpdateDate
+                        from $$ || mdm_db || $$.mst.provider_profile_processing as p
+                        , lateral flatten(input => p.PROVIDER_PROFILE:DEGREE) as json
+                        group by
```
Comment (ASanchezXtillion) [inline migration_original/ODS1Stage/tables/Base/Degree/spu_translated_Degree.sql:40]: not a fan of a distinct with a group by, it's one or the other imo

**Review by ASanchezXtillion — CHANGES_REQUESTED** [review]

File: migration_original/ODS1Stage/tables/Base/Degree/spu_translated_Degree.sql (hunk 1 above)
Comment (carresemata) [reply migration_original/ODS1Stage/tables/Base/Degree/spu_translated_Degree.sql:39]: Changed

**Review by ASanchezXtillion — APPROVED** [review]
LGTM

**Pull request merged**